evap/evaluation/management/commands/send_reminders.py

```diff
@@ -52,7 +52,10 @@ def send_student_reminders():
         for evaluation in Evaluation.objects.filter(
             state=Evaluation.State.IN_EVALUATION, vote_end_date__in=check_dates
         ):
-            recipients.update(evaluation.due_participants)
+            if (
+                datetime.date.today() - datetime.timedelta(days=1) != evaluation.vote_start_datetime.date()
```

Comment:
Two things:
1. I think we should not use a "the start is yesterday" logic, but instead a "this was created less than x days ago". The "yesterday" logic only makes sense because days are discrete, but I think it's more natural to use a less-than check for anything time related
2. We should be able to do this filtering in the database by modifying the arguments to `Evaluation.objects.filter` above. It should be something like `vote_start_datetime__date__ge=datetime.data.today() - datetime.timedelta(days=1)`. Possibly, we need 

Django database functions can be a bit wild when first using them, so please reach out if you need help :)